pkgs/top-level/all-packages.nix

```
@@ -25769,7 +25769,9 @@ with pkgs;
     vmopts = config.jetbrains.vmopts or null;
     jdk = jetbrains.jdk;
   }) // {
-    jdk = callPackage ../development/compilers/jetbrains-jdk {  };
+    sourcebuild = callPackage ../development/compilers/jetbrains-jdk/default.nix {  };
+    prebuild = callPackage ../development/compilers/jetbrains-jdk/prebuild.nix {  };
```

Our usual language for this is "source vs binary" build, so:
```suggestion
    jetbrains-jdk-bin = callPackage ../development/compilers/jetbrains-jdk/bin.nix {  };
```